docs/integrations/destinations/bigquery-denormalized-migrations.md

```diff
@@ -0,0 +1,7 @@
+# BigQuery (denormalized typed struct) Migration Guide
+
+## Upgrading to 2.0.0
+
+`destination-bigquery-denormalized` is being retired in favor of `destination-bigquery`, and is no longer maintained. Please switch to `destination-bigquery`, which will produce similar tables and contains many improvements.  We are retiring `destination-bigquery-denormalized` because it now heavily overlaps with Destinations V2, except for being slower and less reliable. Destinations V2 is now available for BigQuery. To learn more about the feature and speed improvements of Destinations V2, please [click here](https://docs.airbyte.com/release_notes/upgrading_to_destinations_v2)!
```

Comment:
It's not *entirely* a 1-1 switch, there will be some differences (E.g. new table formats), and it will probably be a little slower, but more reliable... We've been in contact with about 1/2 the workspaces using this destination already.